players/models.py

```diff
@@ -1,7 +1,20 @@
+from base64 import urlsafe_b64encode
 from django.contrib.auth.models import User
 from django.db import models
+from os import urandom
 
 
-class Player(models.Model):
-    user = models.OneToOneField(User)
+def generate_auth_token():
+    return urlsafe_b64encode(urandom(16))
+
+
+class Game(models.Model):
+    name = models.CharField(max_length=100)
+    auth_token = models.CharField(max_length=24, default=generate_auth_token)
+
+
+class Avatar(models.Model):
+    owner = models.ForeignKey(User)
+    game = models.ForeignKey(Game)
```

Comment:
Do we want a `unique_together` on owner & game? e.g. 1 avatar per game

I'd say if we wanted to allow multiple avatars for a given user in a game, we should make that a property of the game and then run the code multiple times